Tools/SetupUnityPackage/SetupUnityPackage/Program.cs

```diff
@@ -50,7 +50,7 @@ public static class Program
                 { "lib/netstandard2.0/Realm.dll", "Realm.dll" },
                 { "native/ios/Realm.dll.config", "Realm.dll.config" },
                 { "native/ios/universal/realm-wrappers.framework/realm-wrappers", "iOS/realm-wrappers.framework/realm-wrappers" },
-                { "native/ios/universal/realm-wrappers.framework/Info.plist", "iOS/realm-wrappers.framework/info.plist" },
+                { "native/ios/universal/realm-wrappers.framework/Info.plist", "iOS/realm-wrappers.framework/Info.plist" },
```

Comment:
Oh wow.... I assume that iOS is case sensitive when looking for this. Has iOS always disregarded this? Can the OS make one on the spot if it's missing?

Reply:
It can't - I did get errors about a missing `Info.plist` from the framework.